pkgs/os-specific/bsd/netbsd/default.nix

```diff
@@ -94,6 +86,9 @@ in lib.makeScopeWithSplicing
       platforms = platforms.unix;
       license = licenses.bsd2;
     };
+  } // lib.optionalAttrs stdenv'.hasCC {
+    # TODO should CC wrapper set this?
+    CPP = "${stdenv'.cc.targetPrefix}cpp";
```

Comment:
I'm worried that if we commit this we'll just forget about it and the TODO will be there forever.  How hard would it be to answer this question?

Reply:
@alyssais That's most of my TODOs, I'm sort of OK with that.

I think the issue is that clang didn't install a separate `cpp`, and lots of stuff does `gcc -E`/`clang -E`. I'm recalling when setting `LD` everywhere broke a bunch of things too.

Also, the wrapper might need to be adapted to assume the `-E` flag.